### PR TITLE
Add a note about license for event scripts on Packages tab

### PR DESCRIPTION
**Pull request by yaroslavmo**

### CODE REVIEW

File: app/admin_components/adf-package-manager/views/main.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
             <df-large-help data-options="dfLargeHelp.packageManager"></df-large-help>
             <tabs>
                 <pane title="Export Package">
+                    <span ng-if="!eventScriptEnabled"><b>Note: </b> Upgrade to silver license to be able to export <b><a href="https://wiki.dreamfactory.com/DreamFactory/Features/Scripting/Event_Scripting">Event Scripts</a>.</b></span>
+                    Email <a href="mailto:sales@dreamfactory.com" >sales@dreamfactory.com</a> or call +1-415-612-8958 for pricing information.
```
Comment (wjgilmore) [inline app/admin_components/adf-package-manager/views/main.html]: Please change the phone number to +1 415-993-5877

File: app/admin_components/adf-package-manager/views/main.html [hunk 2]
```diff
@@ -31,6 +33,8 @@
                 </pane>
 
                 <pane title="Import Package">
+                    <span ng-if="!eventScriptEnabled"><b>Note: </b> Upgrade to silver license to be able to import <b><a href="https://wiki.dreamfactory.com/DreamFactory/Features/Scripting/Event_Scripting">Event Scripts</a>.</b></span>
+                    Email <a href="mailto:sales@dreamfactory.com" >sales@dreamfactory.com</a> or call +1-415-612-8958 for pricing information.
```
Comment (wjgilmore) [inline app/admin_components/adf-package-manager/views/main.html]: Please change the phone number to +1 415-993-5877

File: app/admin_components/adf-package-manager/views/main.html [hunk 3]
```diff
@@ -5,6 +5,8 @@
             <df-large-help data-options="dfLargeHelp.packageManager"></df-large-help>
             <tabs>
                 <pane title="Export Package">
+                    <span ng-if="!eventScriptEnabled"><b>Note: </b> Upgrade to silver license to be able to export <b><a href="https://wiki.dreamfactory.com/DreamFactory/Features/Scripting/Event_Scripting">Event Scripts</a>.</b></span>
```
Comment (wjgilmore) [inline app/admin_components/adf-package-manager/views/main.html]: Please change the text to "<b>Note: </b> Upgrade to silver license to be able to export <b><a href="https://wiki.dreamfactory.com/DreamFactory/Features/Scripting/Event_Scripting">Event Scripts</a>.</b>"

Comment (yaroslavmo) [reply app/admin_components/adf-package-manager/views/main.html]: @wjgilmore Do you mean without Email part?

**Pull request merged**